content/copilot/configuring-github-copilot/configuring-github-copilot-in-a-jetbrains-ide.md

```diff
@@ -78,13 +78,13 @@ You can manage advanced settings for {% data variables.product.prodname_copilot
 1. In your JetBrains IDE, click the **File** menu, then click **Settings**.
 1. Under **Languages & Frameworks**, click **{% data variables.product.prodname_copilot %}**.
 1. Edit the settings according to your personal preferences.
-   - To adjust the behaviour and appearance of code suggestions, and whether to automatically check for updates, select or deselect the corresponding checkboxes.
+   - To adjust the behavior and appearance of code suggestions, and whether to automatically check for updates, select or deselect the corresponding checkboxes.
```

Comment:
ℹ️ Note: cSpell defaults to en-US, and I think the docs do too, so I changed the few instances of this that I found.